packages/amplify-graphql-api-construct-tests/src/__tests__/amplify-table.test.ts

```diff
@@ -0,0 +1,41 @@
+import * as path from 'path';
+import { createNewProjectDir, deleteProjectDir } from 'amplify-category-api-e2e-core';
+import { updateCDKAppWithTemplate } from '../update-cdk-app-with-template';
+import { cdkDestroy, initCDKProject, cdkDeploy } from '../commands';
+
+jest.setTimeout(1000 * 60 * 60 /* 1 hour */);
+
+describe('CDK GraphQL Transformer', () => {
+  let projRoot: string;
+  let projFolderName: string;
+
+  beforeEach(async () => {
+    projFolderName = 'cdkamplifytable';
+    projRoot = await createNewProjectDir(projFolderName);
+  });
+
+  afterEach(async () => {
+    try {
+      await cdkDestroy(projRoot, '--all');
+    } catch (_) {
+      /* No-op */
+    }
+
+    deleteProjectDir(projRoot);
+  });
+
+  ['2.80.0'].forEach((cdkVersion) => {
```

Comment:
No need to specify this. You can just write a test, and no need to pass in the default version to `initCDKProject`.